### PR TITLE
Added "go to another checker" functionality

### PR DESCRIPTION
**Pull request by robinpiets**
This PR includes a Modal on the Conclusion component which allows the user to go to another checker. Also, the Modal component is completely refactored to make it more dynamic.

Please make sure:
- [x] to add a label
- [x] to add one or more reviewers
- [x] you followed our checklist for [functional testing](https://github.com/Amsterdam/vergunningcheck/blob/master/TESTING.md)
- [x] you added the necessary documentation (either in the markdown files or inline)
- [x] you added the necessary automated tests

### CODE REVIEW

File: apps/client/src/components/Conclusion.js [hunk 1]
```diff
@@ -113,6 +114,8 @@ const Conclusion = ({ checker, matomoTrackEvent }) => {
             </PrintButton>
           </ComponentWrapper>
         )}
+
+        <ConclusionModal />
```
Comment (svenjens) [inline apps/client/src/components/Conclusion.js]: It's not the conclusion that's in the modal.

**Review by svenjens — CHANGES_REQUESTED** [review]

File: apps/client/src/config/matomo.js [hunk 2]
```diff
@@ -60,11 +63,14 @@ export const eventNames = {
...
   NEWSLETTER: "nieuwsbrief amsterdam.nl",
+  NO_CHOISE_HAS_BEEN_MADE: "er is geen keuze gemaakt",
   PRIVACY: "privacy",
   SAVE_CONCLUSION: "conclusie opslaan",
   TEXT_LINK: "tekst link",
   TO_OLO: "naar het omgevingsloket",
   TWITTER: "twitter",
   WEBARCHIEF: "webarchief",
+  WITH_THE_SAME_ADDRESS: "op hetzelfde adres",
+  WITHOUT_THE_SAME_ADDRESS: "op een andres adres",
```
Comment (svenjens) [inline apps/client/src/config/matomo.js]: typo

File: apps/client/src/components/Modal.test.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  expect(onOpenMock).toHaveBeenCalledTimes(1);
+
+  act(() => {
+    fireEvent.click(queryByTestId(MODAL_CLOSE_BUTTON));
+  });
+
+  expect(queryByTestId(MODAL)).not.toBeInTheDocument();
+});
+
+// @TODO: finish this test
```
Comment (svenjens) [inline apps/client/src/components/Modal.test.js:52]: This test won't be part of this PR right?

File: apps/client/src/components/Conclusion.js (hunk 1 above)
Comment (robinpiets) [reply apps/client/src/components/Conclusion.js]: Check! Thanks

**Review by svenjens — CHANGES_REQUESTED** [review]

File: apps/client/src/config/matomo.js (hunk 2 above)
Comment (robinpiets) [reply apps/client/src/config/matomo.js]: Fixed, thanks

File: apps/client/src/components/Modal.test.js (hunk 3 above)
Comment (robinpiets) [reply apps/client/src/components/Modal.test.js:52]: Depends how much time I have, otherwise I will add it to the list

**Review by svenjens — APPROVED** [review]

**Pull request merged**